setup.py

```diff
@@ -53,12 +53,13 @@
         'Programming Language :: Python :: 3.7',
         'Programming Language :: Python :: 3.8',
         'Programming Language :: Python :: 3.9',
+        'Programming Language :: Python :: 3.10',
+        'Programming Language :: Python :: 3.11',
         'Framework :: Django',
         'Framework :: Django :: 3',
         'Framework :: Django :: 4',
```

Comment:
```suggestion
        'Framework :: Django',
        'Framework :: Django :: 3.2',
        'Framework :: Django :: 4.0',
        'Framework :: Django :: 4.1',
```

Reply:
Thanks @nickmoreton,
This should be fixed on this commit https://github.com/wagtail/wagtail-autocomplete/pull/143/commits/b47237304b10f6119d8ef0094f68d08ee02e0163